speaker/main.go

```diff
@@ -117,7 +116,13 @@ func main() {
 	}()
 	defer level.Info(logger).Log("op", "shutdown", "msg", "done")
 
-	sList, err := speakerlist.New(logger, *myNode, *mlBindAddr, *mlBindPort, *mlSecret, *namespace, *mlLabels, stopCh)
+	mlSecret, err := os.ReadFile("/etc/ml_secret_key/metallb/speaker")
+	if err != nil {
```

Comment:
The secret is optional, so it's fine not to have it. By doing this we are making it mandatory

Reply:
done